src/proto/Cosmos.proto

```diff
@@ -97,7 +144,10 @@ message Message {
         Undelegate unstake_message = 4;
         BeginRedelegate restake_message = 5;
         WithdrawDelegationReward withdraw_stake_reward_message = 6;
-        RawJSON raw_json_message = 7;
+        WasmExecuteContract wasm_execute_contract_message = 7;
+        WasmExecuteContractTransfer wasm_execute_contract_transfer_message = 8;
+        WasmExecuteContractSend wasm_execute_contract_send_message = 9;
+        RawJSON raw_json_message = 10;
```

Comment:
I think rule of thumb for protobuf to never change index number, or it will break things.

Reply:
OK, better to be cautious